pkg/controller/kubelet-config/helpers.go

```diff
@@ -183,7 +183,7 @@ func validateUserKubeletConfig(cfg *mcfgv1.KubeletConfig) error {
 		}
 	}
 
-	resourceFields := []string{"memory.available", "nodefs.available", "nodefs.inodesFree", "imagefs.available"}
+	resourceFields := []string{"memory.available", "nodefs.available", "nodefs.inodesFree", "imagefs.available", "imagefs.inodesFree"}
```

Comment:
Not the most familiar with the flags but reading https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/ there's also a `pid.available`, maybe it's worth adding that as well for futureproof purposes? Or do you foresee that not being used

Reply:
Updated. Thanks